sale_automatic_workflow/models/automatic_workflow_job.py

```diff
@@ -130,23 +130,22 @@ def _validate_pickings(self, picking_filter):
             with savepoint(self.env.cr):
                 self._do_validate_picking(picking, picking_filter)
 
-    def _do_sale_done(self, sale, domain_filter):
-        """Set a sales order to done, filter ensure no duplication"""
+    def _do_sale_lock(self, sale, domain_filter):
```

Comment:
The same about renaming methods. It's not adding too much value in this case, as the goal is the same, and it has been the same name for several versions where done = lock.